apps/demo/src/h5wasm/DropZone.tsx

```diff
@@ -58,12 +57,11 @@ function DropZone(props: Props) {
                 or use a file picker instead
               </button>
             </p>
-            <p
-              className={styles.hint}
-              role={fileRejections.length > 0 ? 'alert' : undefined}
-            >
-              Extensions allowed: <strong>{EXT.join(' ')}</strong>
-            </p>
+            {fileRejections.length > 1 && (
+              <p className={styles.hint} role="alert">
+                Please drop a single file
```

Comment:
I've removed the hint, and I'm now showing an error only when multiple files are dropped.